tracing-opentelemetry/build.gradle

```diff
@@ -28,16 +31,18 @@ dependencies {
 
     testAnnotationProcessor projects.tracingOpentelemetryAnnotation
 
-//    testImplementation(libs.opentelemetry.contrib.aws.resources)
-    testImplementation(libs.opentelemetry.sdk.extension.aws)
+    testImplementation(libs.managed.opentelemetry.contrib.aws.resources)
+//    testImplementation(libs.opentelemetry.sdk.extension.aws)
 
-    testImplementation mn.reactor
-    testImplementation mn.micronaut.reactor.http.client
+    testImplementation mnReactor.micronaut.reactor
```

Comment:
Nitpick: I think this can stay as `mn.reactor`

Reply:
it has to be `mnReactor.micronaut.reactor` because some code is moved from micronaut-core to micronaut-reactor